_rules/form-field-non-empty-accessible-name-e086e5.md

```diff
@@ -45,10 +45,10 @@ _There are currently no assumptions_
 
 The list of roles in the applicability is derived by taking all the roles from [WAI-ARIA Specifications](#wai-aria-specifications) that:
 
-- have [semantic roles][] that inherit from the [abstract](https://www.w3.org/TR/wai-aria/#abstract_roles) `input` or `select` role, and
+- have [semantic roles][] that inherit from the [abstract](https://www.w3.org/TR/wai-aria/#abstract_roles) `input`, `select` or `menuitem` role, and
 - do not have a [required context](https://www.w3.org/TR/wai-aria/#scope) role that itself inherits from one of those roles.
 
-Note that this rule does not test other control-like roles such as `button` and `menuitem`, because these do not inherit from `input` or `select`. These should be tested separately.
+Note that this rule does not test other control-like roles such as `button`, because these do not inherit from `input` or `select`. These should be tested separately.
```

Comment:
I think you can leave this as is. `menuitem` is still not applicable. Things that inherit from menuitem are.

```suggestion
Note that this rule does not test other control-like roles such as `button` and `menuitem`, because these do not inherit from `input` or `select`. These should be tested separately.
```